tasks/documentation.js

```diff
@@ -25,23 +25,6 @@ const srcPath = path.join(projectDir, 'src');
 
 const BASE_URL = 'https://opensource.adobe.com/spectrum-web-components/';
 
-const extractComponentDocumentation = () => {
-    return exec(
-        `node "${path.join(
-            projectDir,
-            'documentation/scripts/extractComponentAPIDocs.js'
```

Comment:
The work that was being done in `documentation/scripts/extractComponentAPIDocs.js` is no longer compatible with the most recent version of `web-component-analyzer`, so I took it out of the workflow and replaced it with the component based JSON interpolation above.